app/src/main/java/com/infomaniak/swisstransfer/ui/theme/Theme.kt

```diff
@@ -62,4 +82,7 @@ object SwissTransferTheme {
     val typography: CustomTypography
         @Composable
         get() = LocalCustomTypography.current
+    val colors: CustomColors
```

Comment:
Add delegate material colors

Reply:
We can't use delegation on composable method but I added the property anyway